api/organisations/models.py

```diff
@@ -363,6 +365,11 @@ def get_subscription_metadata(self) -> BaseSubscriptionMetadata:
             # plan.
             return FREE_PLAN_SUBSCRIPTION_METADATA
 
+        if self.plan == SCALE_UP and self.subscription_id == "test_subscription_id":
```

Comment:
With regards to obtaining subscription data, this is the process followed by the e2e test:

https://github.com/Flagsmith/flagsmith/blob/7456fa320b9001613cf50145a53b0a957d3c9197/api/organisations/models.py#L408-L416

We need to support multiple projects for the test that creates a new project based on permissions.

Reply:
That code is not for testing the API, it's for testing the E2E tests in the FE that call the get-subscription-metadata endpoint. Among the response values, it includes the number of projects we are allowed to have. Without that code, the response returns projects: 1, which prevents the frontend from creating a new project to test the CREATE_PROJECT permission.

Reply:
Another option would be to make sure that we're testing against a container that returns `is_enterprise() == True` ? I think that would just be a case of adding a volume mapped to `/app/ENTERPRISE_VERSION` ? 

Otherwise, we definitely need to think of a better way to achieve this. The approach here is far too hacky - relying on a specific string. Can't we use the E2E_TEST_TOKEN somehow for example?